src/main/java/com/example/icecreamup/resources/application.properties

```diff
@@ -1 +0,0 @@
-
```

Comment:
This file was redundant so I removed it.  There is already an application.properties file that is properly set up in the project directory.